keymap/vim.js

```diff
@@ -411,11 +411,11 @@
       cfg = cfg || {};
       var scope = cfg.scope;
       if (!option) {
-        throw Error('Unknown option: ' + name);
+        return new Error('Unknown option: ' + name);
```

Comment:
I agree -- exception control flow, especially in JavaScript, is quite a lot harder to deal with and easier to mess up, so a regular return seems sensible.